core/src/main/java/org/apache/metamodel/schema/MutableColumn.java

```diff
@@ -70,9 +71,10 @@ public MutableColumn(String name, ColumnType type, Table table, int columnNumber
     }
 
     public MutableColumn(String name, ColumnType type, Table table, int columnNumber, Integer columnSize,
-            String nativeType, Boolean nullable, String remarks, boolean indexed, String quote) {
+                         Integer decimalDigits, String nativeType, Boolean nullable, String remarks, boolean indexed, String quote) {
```

Comment:
Same comment as for the `ImmutableColumn` constructor change, by adding a parameter to this constructor, the code will cause compilation errors in other project using this constructor. Can you please add an extra constructor with the signature of the original version of this constructor, which invokes this constructor with a default value for `decimalDigits`?

Reply:
Fixed!

Reply:
This looks good now.